services/data/src/ugdata/aws.py

```diff
@@ -51,6 +51,12 @@ def lambda_handler(event, context):
     if "Event" in event and event["Event"] == "s3:TestEvent":
         return "Test event received"
 
+    if "Records" not in event:
+        # FIXME: This should use a real logger
+        print("Records missing from event")
+        print(event)
+        return ""
```

Comment:
Would it make sense to return something like `Records missing from event` here? That could be deferred for the future.

Reply:
I think so. I don’t know what Lambda is doing with the return values, since we don’t have this hooked up to API Gateway. I figured an empty string was the closest thing to a no-op and thus less likely to cause more exceptions than non-empty string.